src/Domain/CampaignRepository.php

```diff
@@ -164,16 +233,6 @@ protected function doUpdateFromSf(SalesforceReadProxy $proxy, bool $withCache):
             throw new DomainCurrencyMustNotChangeException();
         }
 
-        $charity = $this->pullCharity(
-            salesforceCharityId: $campaignData['charity']['id'],
-            charityName: $campaignData['charity']['name'],
-            stripeAccountId: $campaignData['charity']['stripeAccountId'],
-            giftAidOnboardingStatus: $campaignData['charity']['giftAidOnboardingStatus'],
-            hmrcReferenceNumber: $campaignData['charity']['hmrcReferenceNumber'],
-            regulator: $campaignData['charity']['regulatorRegion'],
-            regulatorNumber: $campaignData['charity']['regulatorNumber'],
-        );
-
```

Comment:
we know this is always going to update the existing charity that's attached to the campaign, not pull a new one.